.github/workflows/encryption-pvc-kms-ibm-kp/action.yml

```diff
@@ -24,8 +21,10 @@ runs:
 
     - name: setup cluster resources
       uses: ./.github/workflows/canary-test-config
-      with:
-        github-token: ${{ inputs.github-token }}
+
+    - name: set Ceph version in CephCluster manifest
+      shell: bash --noprofile --norc -eo pipefail -x {0}
```

Comment:
in `canary-integration-test.yaml`  change to use a custom ceph image is not added for this particular test. Adding here makes more sense for `encryption-pvc-kms-ibm-kp` test so it is added here.

Reply:
My question is why the bash command is needed here (line 26)?

Reply:
oh, the bash command is added here because it is a [composite file](https://docs.github.com/en/actions/creating-actions/creating-a-composite-action#creating-an-action-metadata-file) which doesn't have direct access to the bash